### PR TITLE
added mudirac recipe

### PR DESCRIPTION
**Pull request by subindev-d**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by subindev-d** [comment]
Hi @patrick-austin , pleae comment here.

**Comment by patrick-austin** [comment]
👍🏼

**Comment by subindev-d** [comment]
@conda-forge/help-c-cpp ready for review!

**Comment by subindev-d** [comment]
@@conda-forge/staged-recipes, ready for review

**Comment by subindev-d** [comment]
@conda-forge-admin, please ping @conda-forge/help-c-cpp

File: recipes/mudirac/build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+mkdir -p build
+cd build
+cmake $SRC_DIR
```
Comment (xhochy) [inline recipes/mudirac/build.sh:4]:
```suggestion
cmake $CMAKE_ARGS $SRC_DIR
```

File: recipes/mudirac/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+make
```
Comment (xhochy) [inline recipes/mudirac/build.sh:5]:
```suggestion
make -j $CPU_COUNT
```

File: recipes/mudirac/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+ctest
+mkdir -p $PREFIX/bin/
+cp -r bin/mudirac $PREFIX/bin/
```
Comment (xhochy) [inline recipes/mudirac/build.sh:8]: Doesn't this have a `make install` command?

File: recipes/mudirac/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{% set name = "mudirac" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: mudirac
+  version: 1.0.1
```
Comment (xhochy) [inline recipes/mudirac/meta.yaml:7]:
```suggestion
  version: {{ version }}
```

File: recipes/mudirac/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  url:  https://github.com/muon-spectroscopy-computational-project/mudirac/archive/refs/tags/1.0.1.tar.gz
```
Comment (xhochy) [inline recipes/mudirac/meta.yaml:10]:
```suggestion
  url:  https://github.com/muon-spectroscopy-computational-project/mudirac/archive/refs/tags/{{ version }}.tar.gz
```

File: recipes/mudirac/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 801cf46c71726265f988e984f539354c6fbab070dc101090009ee92d09ce0faa
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
```
Comment (xhochy) [inline recipes/mudirac/meta.yaml:16]: Why are they failing? Seems like a simple recipe and this it would be nice to include them.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/mudirac/build.sh (hunk 3 above)
Comment (subindev-d) [reply recipes/mudirac/build.sh:8]: it doesnt have `make install` its either `make mudirac` which only builds the package and skip tests and etc.

File: recipes/mudirac/meta.yaml (hunk 6 above)
Comment (subindev-d) [reply recipes/mudirac/meta.yaml]: now not skipping for osx, which works fine. As the mudirac package is using `make` build it is not building against windows. Modifying the build system suitable for windows can be considered in the future.

Comment (subindev-d) [reply recipes/mudirac/meta.yaml:16]: tried osx, but failed due to one of the library dependencies https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=916285&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1414

Comment (subindev-d) [reply recipes/mudirac/meta.yaml:16]: will conder OSx in in the future releases.

**Comment by xhochy** [comment]
For OSX, it would be probably sufficient to add a `conda_build_config.yaml` with the following content:

```
c_stdlib_version:  # [osx and x86_64]
  - 10.12  # [osx and x86_64]
```

**Comment by xhochy** [comment]
Seems like there are more issues, happy to proceed without `osx` then.

**Comment by subindev-d** [comment]
> Seems like there are more issues, happy to proceed without `osx` then.

Thanks I will revert the changes done for OSx.

**Review by xhochy — APPROVED** [review]

**Pull request merged**